### PR TITLE
Feature/address lookup

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
Refactored the web form components a bit to make it scalable.
Added Address Finder

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] i18n

### CODE REVIEW

File: src/JSONFormData/FirstTimeApplication.js [hunk 1]
```diff
@@ -1,159 +1,187 @@
...
-      'ma': {
-        'text': 'Ko wai tō wāhi noho?'
+      'mi': {
+        'text': 'Ko wai tō ingoa?'
       }
     },
-    'type': 'text',
-    'name': 'address',
-    'isRequired': true
+    'isRequired': true,
```
Comment (Br3nda) [inline src/JSONFormData/FirstTimeApplication.js]: how about "required", without the "is".
the "is" is implied.

File: src/JSONFormData/FirstTimeApplication.js [hunk 2]
```diff
@@ -1,159 +1,187 @@
...
+    'component': RenderRadio
   },
   {
-    'question': {
+    'label': {
       'en': {
-        'text': 'Income'
+        'text': 'Super Annuation'
```
Comment (Br3nda) [inline src/JSONFormData/FirstTimeApplication.js]: I just checked, and it's one word: Superannuation.

File: src/styles/CheckboxGroup.css.map [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
...CnB,aAAa,EAAE,CAAC;AAGlB,4BAAa;EACX,MAAM,EAAE,CAAC;AAGX,oCAAqB;EACnB,QAAQ,EAAE,QAAQ;EAClB,IAAI,EAAE,OAAO;AAGf,4CAA6B;EAC3B,QAAQ,EAAE,QAAQ;EAClB,MAAM,EAAE,OAAO;EACf,OAAO,EAAE,IAAI;EACb,WAAW,EAAE,MAAM;EACnB,YAAY,EAAE,IAAI;EAClB,aAAa,EAAE,IAAI;EACnB,UAAU,EAAE,IAAI;EAChB,SAAS,EAAE,IAAI;EACf,gBAAgB,EAAE,OAAO;EACzB,OAAO,EAAE,IAAI;EACb,aAAa,EAAE,IAAI;EACnB,kDAAQ;IACN,gBAAgB,EAAE,OAAmB;EAGvC,yBAAyB;IAhB3B,4CAA6B;MAiBzB,KAAK,EAAE,KAAK;EAGd,iDAAK;IACH,WAAW,EAAE,IAAI;IACjB,WAAW,EAAE,GAAG;EAGlB,yDAAa;IACX,OAAO,EAAE,EAAE;IACX,MAAM,EAAE,iBAAyB;IACjC,UAAU,EA5DC,OAAO;IA6DlB,KAAK,EAAE,IAAI;IACX,MAAM,EAAE,IAAI;IACZ,aAAa,EAAE,IAAI;IACnB,QAAQ,EAAE,QAAQ;IAClB,GAAG,EAAE,GAAG;IACR,IAAI,EAAE,IAAI;IACV,SAAS,EAAE,gBAAgB;EAG7B,oEAAwB;IACtB,gBAAgB,EAvEL,OAAO;IAwElB,YAAY,EAxED,OAAO;EA2EpB,mEAAuB;IACrB,WAAW,EAAE,aAAa;IAC1B,SAAS,EAAE,IAAI;IACf,OAAO,EAAE,OAAO;IAChB,QAAQ,EAAE,QAAQ;IAClB,IAAI,EAAE,IAAI;IACV,GAAG,EAAE,GAAG;IACR,SAAS,EAAE,gBAAgB;IAC3B,OAAO,EAAE,CAAC;IACV,KAAK,EAnFH,IAAI;;AA0FV,iBAAG;EACD,UAAU,EAAE,IAAI;EAChB,aAAa,EAAE,IAAI;AAErB,oBAAM;EAAE,OAAO,EAAE,eAAe;AAChC;qBACO;EACL,aAAa,EAAE,IAAI;EACnB,OAAO,EAAE,YAAY;AAEvB,iBAAG;EAAC,YAAY,EAAE,IAAI;;AAGxB,YAAa;EACX,YAAY,EAAE,CAAC;EACf,eAAG;IACD,UAAU,EAAE,IAAI",
```
Comment (Br3nda) [inline src/styles/CheckboxGroup.css.map]: should we be checking these files in???

File: src/styles/RadioGroup.css [hunk 4]
```diff
@@ -43,9 +43,16 @@
       text-align: center;
       font-size: 16px;
       padding: 8px 20px; }
-  @media (max-width: 320px) {
-    .radio-group .checkbox-radio span {
-      font-size: 15px;
-      padding: 10px 12px; } }
+  .radio-group .checkbox-radio span {
+    font-size: 15px;
+    padding: 12px 9px; }
```
Comment (Br3nda) [inline src/styles/RadioGroup.css]: the curly brackets are a bit hard to read like this.

Comment (dlouise64) [reply src/styles/RadioGroup.css]: @Br3nda Yep thats the generated version after running `npm run sass`.

**Pull request merged**